about/about-ebpf.md

```diff
@@ -114,28 +115,28 @@ While the eBPF dataplane has some features that the standard Linux dataplane lac
 | Direct Server Return      | Not supported                           | Supported (requires compatible underlying network) |
```

Comment:
Referring to the "Throughput" factor, isn't is better to use Gbps rather than GBit. It's not part of the PR, but it got my attention! Normally, for throughput an unit of speed is needed.